src/SignIn/SignInScreen.js

```diff
@@ -41,46 +41,80 @@ const styles = StyleSheet.create({
 export default function SignInScreen({ navigation }) {
   const [firstName, setFirstName] = useState('');
   const [lastName, setLastName] = useState('');
+  const [organization, setOrganization] = useState('');
   const [email, setEmail] = useState('');
   const [password, setPassword] = useState('');
   const [number, setNumber] = useState('');
+  const [busPhone, setBusPhone] = useState('');
   const [address, setAddress] = useState('');
   const [hidePass, setHidePass] = useState(true);
 
   const lastNameInput = useRef();
+  const organizationInput = useRef();
   const emailInput = useRef();
   const passwordInput = useRef();
   const numberInput = useRef();
+  const busPhoneInput = useRef();
   const addressInput = useRef();
 
+  const buttonClickListener = () =>{
+    // console.log("firstname")
+    // console.log(firstName)
+    if (firstName == ""){
+      Alert.alert("Please enter the firstName to proceed")
+    }
+  }
+
   const handleSignUp = () => {
-    base('Users').create([
-      {
-        fields: {
-          email,
-          password,
-          'first name': firstName,
-          'last name': lastName,
-          'phone number': number,
-          address,
+    if (firstName == ""){
```

Comment:
This looks good but I don't think it passes eslint, make sure estlint is working in your env and don't push before checking that everything passes please!